test/gtest/cache.cpp

```diff
@@ -32,6 +32,8 @@
 #include "test.hpp"
```

Comment:
For some reason I cannot see this file in https://github.com/ROCmSoftwarePlatform/MIOpen/tree/sl/google_test/test/gtest

Reply:
Hey @bghimireamd , thank you for the comments. this header file is actually in an upper level :)
https://github.com/ROCmSoftwarePlatform/MIOpen/tree/sl/google_test/test

Reply:
Oh! sorry I meant to say I cannot see file `test/gtest/cache.cpp` in https://github.com/ROCmSoftwarePlatform/MIOpen/tree/sl/google_test/test/gtest

Reply:
I see. I have checked out a fresh copy and switch to sl/test_cache branch. Seems it's there.
$ git clone https://github.com/ROCmSoftwarePlatform/MIOpen.git
$git switch sl/test_cache
$ls
CMakeLists.txt  cache.cpp  hello_test.cpp

Thanks